server/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -59,7 +59,7 @@ public interface ReadOnlyApplicantProgramService {
    * Get the program block with the lowest index that has missing answer data if there is one.
    * Static questions are marked as incomplete.
    */
-  Optional<Block> getFirstIncompleteBlock();
```

Comment:
I renamed this as it seemed like for callers the implication was subtle but very important to know in doing the right thing.

Reply:
ack